### PR TITLE
[14.0][MIG] stock_report_quantity_by_location: migration from 13.0 to 14.0

### PR DESCRIPTION
**Pull request by AlvaroTForgeFlow**
@ForgeFlow

### CODE REVIEW

**Review by ChrisOForgeFlow — APPROVED** [review]
+1

File: stock_report_quantity_by_location/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_report_quantity_by_location_prepare,access_stock_report_quantity_by_location_prepare,model_stock_report_quantity_by_location_prepare,base.group_user,1,1,1,1
+access_stock_report_quantity_by_location,access_stock_report_quantity_by_location,model_stock_report_quantity_by_location,base.group_user,1,1,1,1
```
Comment (LoisRForgeFlow) [inline stock_report_quantity_by_location/security/ir.model.access.csv]: Change the group to stock.group_stock_user (Inventory User) and the permission to 1,1,1,0 (standard permission in any wizard in the core).

Comment (AlvaroTForgeFlow) [reply stock_report_quantity_by_location/security/ir.model.access.csv]: Done.

**Review by jonerikceberio — APPROVED** [review]
I had done the same migration, with the only difference of the order of the lines in the manifest... so LGTM

**Comment by rousseldenis** [comment]
@AlvaroTForgeFlow Could you rebase ?

**Comment by JordiMForgeFlow** [comment]
Hi all,

I am suggesting some changes in V13 (https://github.com/OCA/stock-logistics-reporting/pull/204). Maybe you agree with what I am suggesting :)

**Comment by rousseldenis** [comment]
@AlvaroTForgeFlow Waht's the status of this ?

Could be great to merge this before 15.0 one. Could you include 13.0 changes #204 ?

**Comment by ChrisOForgeFlow** [comment]
@rousseldenis @JordiMForgeFlow #204 was included on this PR, thanks!

**Pull request merged**